commons/genomics.py

```diff
@@ -21,23 +21,42 @@ class Variant(JSONifiable):
     reference = attr.ib(validator=instance_of(str))
     alternate = attr.ib(validator=instance_of(str))
 
+    PARSER = re.compile('''^(chr)?
+                            (?P<chromosome>( M | MT | X | Y |
+                                             1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 
+                                             11 | 12 | 13 | 14 | 15 |16 | 17 | 18 | 19 | 20 | 
+                                             21 | 22 | 23 | 24 | 25))
+                                             
+                            (?P<separator>[_:/])
+                            
+                            (?P<position>\d+)
+
+                            (?P=separator)
+                            
+                            (?P<reference>( \<[^\>]{1,998}\>
```

Comment:
Currently our longest alt allele is 421 nucleotides long, and this may grow once in the future when we get a new imputation panel. Depending on the database 1000 is probably indeed a good max length, given that we will also use this for other than FinnGen data.